### ISSUE TITLE
unsubscribe me from distribution list

### ISSUE THREAD

**Issue by eebeeDEV** [FortAwesome/Font-Awesome#11325]
I've ordered font awesome and since then i'm getting tens of emails per day regarding font awesome.
I would like to unsubscribe but can't see anywhere how to do that, not on both your websites (.io & .com)
Can you please tell me where I can do that or unsubscribe from your list?

Thank you,
Eric

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Does this link at the bottom of the e-mail work?
![image](https://user-images.githubusercontent.com/556268/28619276-dd64167c-7207-11e7-96c1-0f25d7c5e4f4.png)

**Comment by grifferz** [comment]
Your emails have a text/plain version and a text/html version and this unsubscribe information is not in the text/plain part. I actually found this issue while searching for how to unsubscribe. I didn't think to check the text/html version. So, you should add the unsubscribe link to the text/plain part too.

**Comment by tagliala** [comment]
Thanks for the detailed information

@robmadole could you please check?

**Comment by robmadole** [comment]
@grifferz great catch! thank you. We'll look into that.

**Comment by gwijnja** [comment]
I'm currently receiving daily email updates about the Web Awesome Kickstarter. And again there's no unsubscribe link at the bottom. They do not originate from Kickstarter, so I can not unsubscribe via their website. So they have to come from you directly. The word 'unsubscribe' does not appear on your support/faq page. When googling for "font awesome mailinglist", all I get are mailinglist icons. So it is completely unclear how to remove yourself from your mailinglist. PLEASE add an unsubscribe link to your emails.

![image](https://github.com/FortAwesome/Font-Awesome/assets/2666867/deb25c8c-bfcb-4917-9fd8-64e0d487b0e1)

**Comment by robmadole** [comment]
@gwijnja we are really careful to include those in all of our email. Can you help us figure out why you can't see it?

What was the title of the email?

Here is what I see:

<img width="813" alt="oRIbq3Ga@2x" src="https://github.com/FortAwesome/Font-Awesome/assets/132895/c9265cec-8985-4c03-8791-5d9abffcfb83">

**Comment by sensibleworld** [comment]
Thanks for the heads-up, folks! It's against our nature to send so many emails, but it’s really the best way we have to get the word out about Web Awesome. (And KS gives us no way to get folks email addresses to exclude backers until the campaign is over.)

But unsubscribing should also be quick and easy! That's definitely not our intent.

**Comment by gwijnja** [comment]
> @gwijnja we are really careful to include those in all of our email. Can you help us figure out why you can't see it?
> 
> What was the title of the email?

That's interesting. I'm using [Postbox](https://www.postbox-inc.com/) to read my mail. Even when I select everything (ctrl+a) it doesn't show up, so it's not a matter of light gray text on a light gray background or so. And I do see the link in the mail source. So I suspect there's something in the HTML that is preventing it from being displayed. I'll have a closer look.

The title is "🎉 Thank You! We've hit a $200K milestone! 🎉"

**Comment by tagliala** [comment]
> The title is "🎉 Thank You! We've hit a $200K milestone! 🎉"

FWIW, I have that link in the  very same email (both gmail and thunderbird)

![image](https://github.com/FortAwesome/Font-Awesome/assets/556268/e195af3a-3e35-4714-a616-c5d19550bdfd)

**Comment by robmadole** [comment]
I downloaded Postbox and this is what I'm seeing:

<img width="828" alt="YNHFu975@2x" src="https://github.com/FortAwesome/Font-Awesome/assets/132895/9b58f114-90d1-40ff-8c9f-66870e09238e">

But if I look at the message source I see this:

![3EPgwnmR@2x](https://github.com/FortAwesome/Font-Awesome/assets/132895/df2cf3b5-286c-4855-812f-4a067bc09031)

Seems like it's there but Postbox isn't rendering it? @gwijnja any thoughts?

**Comment by gwijnja** [comment]
Yes, I'm slowly digging my way through the maze. But frankly speaking it's such a mess... For a company that specializes in neat HTML and CSS, with the new Web Awesome project, the HTML in this email is horrendous. You're probably using some kind of tool to prepare these emails, but it's awful.

Some things stand out immediately: the table in a table in a table in a table. I haven't seen constructions like this since 2005. There must be a better way, right?
![image](https://github.com/FortAwesome/Font-Awesome/assets/2666867/2459f982-145e-467e-a5d9-4a82f9b93efb)

And I see the center-tag still being used:
![image](https://github.com/FortAwesome/Font-Awesome/assets/2666867/868c0b34-3062-495c-9d7d-04948b837ef4)

That tag has been deprecated since the introduction of HTML5, 15 years ago. Please use CSS instead.

Here is even an entire table with bloated mark-up and everything, just to print a space character!
![image](https://github.com/FortAwesome/Font-Awesome/assets/2666867/f2ca69ec-2bd5-406a-ba8b-bcb37dbea735)

The address footer is not in a td-tag. So under the table tag comes a tbody, tr, and then a p tag. There should be a td (table data) between the tr and the p.
![image](https://github.com/FortAwesome/Font-Awesome/assets/2666867/b55ee369-b517-4d13-86ff-10344eeb5c5e)

Anyway, back to the original issue. This part is a little strange:
![image](https://github.com/FortAwesome/Font-Awesome/assets/2666867/38ec9ce5-ea98-456b-803f-1004cd3f9f84)

(I collapsed the space-in-the-table part.) The entire mail body is placed between the (deprecated) center tags. You can still see the address part at the top, then the whole table structure is torn down, the center tag is closed, as if the email is done. Next you see a list of nbsp characters, and then the unsubscribe part is added. It should probably have been part of the rest of the mail, in the table structure somewhere between the center tags.

But ok, it shouldn't matter, it should still be visible.

@robmadole You can save the email as an .eml file, open it in an editor and then carefully start cutting away parts, and reloading the file in Postbox. That's what I'm going to do now. Because I do like this puzzle. :)

**Comment by gwijnja** [comment]
Solved!

The main table (at the top of the HTML) has an inline stylesheet that stretches the table across the whole width and height of the viewport:

![image](https://github.com/FortAwesome/Font-Awesome/assets/2666867/cbbc33b9-2464-4963-b5e7-6f14721e64b3)

The email is still using the old method of tables to structure a website. (A modern alternative would be to use a grid system, like CSS Grid/Flexbox.) Setting the width and height to 100% causes the table to stretch over the entire email.

But the unsubscribe link is not in the main table. It is in a separate table, that comes **after** the table that covers the entire viewport. (See the last screenshot in my previous comment.) The big one overlaps the small unsubscribe table. So it's there, but it's **behind** the big table. When I move the unsubscribe table to just before the `</center>` closing tag, the link appears:

![image](https://github.com/FortAwesome/Font-Awesome/assets/2666867/a8058a81-a4ca-436f-b1d4-068755980f8a)

Setting the table height to for example 90% also works. But moving the table to just before the `</center>` tag is probably the best way to fix this issue.

**Comment by sensibleworld** [comment]
Nice sleuthing! It's high time we took a second look at how these emails are built. If memory serves, a lot of the hackiness had to be in place so that folks using Microsoft Outlook could actually see the content. Every other client let us be clean and modern.

All that said! If you wanna unsubscribe, drop us an email directly to hello [at] fontawesome.com (feel free to mention me) and we'll make sure you get unsubscribed.

**Comment by gwijnja** [comment]
> Nice sleuthing! It's high time we took a second look at how these emails are built. If memory serves, a lot of the hackiness had to be in place so that folks using Microsoft Outlook could actually see the content. Every other client let us be clean and modern.

Yeah I noticed the `<--[if mso]>` hacks as well. Outlook actually uses Word to render HTML, iirc. And it needs a lot of exceptions.

> All that said! If you wanna unsubscribe, drop us an email directly to hello [at] fontawesome.com (feel free to mention me) and we'll make sure you get unsubscribed.

Thanks! But the link works fine with the fix. Good luck with the project, it looks very promising! 👍

**Comment by sensibleworld** [comment]
Closing this out... but I wanted to note that there is also now a direct setting within your Font Awesome account to turn marketing emails on/off.
